### PR TITLE
UCP: additionalPrinterColumns tidbautoscaler

### PR DESCRIPTION
**Pull request by shonge**
### What problem does this PR solve? 
UCP: #1820 
### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)
```
kubectl -n t1 apply -f examples/basic/tidb-cluster.yaml
kubectl -n t1 apply -f examples/basic/tidb-autoscaler.yaml
kubectl -n t1 get ta 
NAME       TIDB-MAXREPLICAS   TIDB-MINREPLICAS   TIKV-MAXREPLICAS   TIKV-MINREPLICAS
basic-autosacler   5                  2                  5                  2
```

Code changes

 - Has Go code change

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: examples/basic/tidb-autoscaler.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  name: basic-autosacler
+spec:
+  cluster:
+    name: basic
+  tidb:
+    maxReplicas: 5
+    minReplicas: 2
+  tikv:
+    maxReplicas: 5
+    minReplicas: 2
```
Comment (Yisaer) [inline examples/basic/tidb-autoscaler.yaml]: I suggest revert this change. The rest LGTM.

**Comment by Yisaer** [comment]
@cofyc @aylei  PTAL

File: pkg/util/crdutil.go [hunk 2]
```diff
@@ -171,6 +171,33 @@ var (
...
+	autoScalerTiDBMaxReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiDB-MaxReplicas",
+		Type:        "integer",
+		Description: "The max replicas number of TiDB cluster",
+		JSONPath:    ".spec.tidb.maxReplicas",
+	}
+	autoScalerTiDBMinReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiDB-MinReplicas",
+		Type:        "integer",
+		Description: "The Min replicas number of TiDB cluster",
```
Comment (cofyc) [inline pkg/util/crdutil.go]:
```suggestion
		Description: "The minimal replicas of TiDB",
```

File: pkg/util/crdutil.go [hunk 3]
```diff
@@ -171,6 +171,33 @@ var (
...
+		Name:        "TiKV-MinReplicas",
+		Type:        "integer",
+		Description: "The Min replicas number of TiKV cluster",
+		JSONPath:    ".spec.tikv.minReplicas",
+	}
+	// TODO add The current replicas number of TiDB cluster
+	autoScalerTiDBMaxReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiDB-MaxReplicas",
+		Type:        "integer",
+		Description: "The max replicas number of TiDB cluster",
```
Comment (cofyc) [inline pkg/util/crdutil.go]:
```suggestion
		Description: "TThe maximal replicas of TiDB",
```

File: pkg/util/crdutil.go [hunk 4]
```diff
@@ -171,6 +171,33 @@ var (
...
+	autoScalerTiKVMaxReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiKV-MaxReplicas",
+		Type:        "integer",
+		Description: "The max replicas number of TiKV cluster",
+		JSONPath:    ".spec.tikv.maxReplicas",
+	}
+	autoScalerTiKVMinReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiKV-MinReplicas",
+		Type:        "integer",
+		Description: "The Min replicas number of TiKV cluster",
```
Comment (cofyc) [inline pkg/util/crdutil.go]:
```suggestion
		Description: "The minimal replicas of TiKV",
```

File: pkg/util/crdutil.go [hunk 5]
```diff
@@ -171,6 +171,33 @@ var (
 		Priority:    1,
 		JSONPath:    ".status.phase",
 	}
+	autoScalerPrinterColumns []extensionsobj.CustomResourceColumnDefinition
+	// TODO add The current replicas number of TiKV cluster
+	autoScalerTiKVMaxReplicasColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "TiKV-MaxReplicas",
+		Type:        "integer",
+		Description: "The max replicas number of TiKV cluster",
```
Comment (cofyc) [inline pkg/util/crdutil.go]:
```suggestion
		Description: "The maximal replicas of TiKV",
```

**Comment by cofyc** [comment]
the CRD yaml file must be updated

**Comment by Yisaer** [comment]
@shonge   Plz address the comment. You could update the crd by `hack/update-crd-groups.sh`

**Comment by shonge** [comment]
> @shonge Plz address the comment. You could update the crd by `hack/update-crd-groups.sh`

Done.
